sdk/core/azure-core/src/main/java/com/azure/core/http/rest/PagedFlux.java

```diff
@@ -36,107 +33,34 @@
  * @see Page
  * @see Flux
  */
-public class PagedFlux<T> extends Flux<T> {
-    private final Supplier<Mono<PagedResponse<T>>> firstPageRetriever;
-    private final Function<String, Mono<PagedResponse<T>>> nextPageRetriever;
+public class PagedFlux<T> extends PagedFluxBase<T, PagedResponse<T>> {
```

Comment:
This class is delegating everything to the superclass. The only purpose of this class is to declare the PagedResponse type. @JonathanGiles any suggestions/known patterns to get around this?

Reply:
we need to be careful with changes to this calss is because SDKs are using this.
Cannot remove/make major changes

Reply:
I think what we have now (in this PR) is ok. I don't think there is a better option.